src/main/java/de/tum/in/www1/artemis/web/rest/OrganizationResource.java

```diff
@@ -69,14 +72,14 @@ public OrganizationResource(OrganizationService organizationService, Organizatio
     }
 
     /**
-     * DELETE /organizations/course/:courseId/organization/:organizationId :
+     * DELETE organizations/course/:courseId/organization/:organizationId :
      * Remove a course from an organization
      *
      * @param courseId the id of the course to remove
      * @param organizationId the id of the organization from with the course should be removed
      * @return empty ResponseEntity with status 200 (OK), or 404 (Not Found) otherwise
      */
-    @DeleteMapping("/organizations/course/{courseId}/organization/{organizationId}")
+    @DeleteMapping("organizations/course/{courseId}/organization/{organizationId}") // TODO: should be organizations/{organizationId}/courses/{courseId}
```

Comment:
You have changed all other REST Calls, so I suggest you improve this one as well